Dockerfile.precache

```diff
@@ -6,5 +6,8 @@ COPY pre-cache/release \
      pre-cache/parse_index.py \
      pre-cache/pull \
      pre-cache/precache.sh \
+     pre-cache/copy-env.sh \
      /opt/precache
-ENV PATH="/opt/precache:${PATH}"
+COPY pre-cache/storage.conf /etc/containers/
+ENV PATH="/opt/precache:/host/usr/bin:/host/usr/libexec:${PATH}"
+ENV LD_LIBRARY_PATH="/host/usr/lib64"
```

Comment:
Yes...
The operation here is divided into two phases: pull spec creation and actual pulling of the images. The latter one is done using "chroot /host", so no problem. 
But the first part can't be (or difficult to) run "chrooted" because of selinux issue I had when running the opm binary. I run opm (that calls podman off the host) inside the container by adding /host/usr/bin/ to the PATH. Podman is using lots of libraries, so I set their location (/host/usr/lib64 as it is seen from the container) to the LD_LIBRARY_PATH.